cmd/attack/disk.go

```diff
@@ -70,14 +70,19 @@ func NewDiskWritePayloadCommand(dep fx.Option, options *core.DiskCommand) *cobra
 	}
 
 	cmd.Flags().StringVarP(&options.Size, "size", "s", "",
-		"'size' specifies how many data will fill in the file path with unit MB.")
-	cmd.Flags().StringVarP(&options.Path, "path", "p", "/dev/null",
+		"'size' specifies how many units of data will write into the file path.\n"+
+			"'unit' specifies the unit of data, support c=1, w=2, b=512, kB=1000, K=1024, MB=1000*1000,"+
+			"M=1024*1024, , GB=1000*1000*1000, G=1024*1024*1024 BYTES\n"+
+			"example : 1M | 512kB")
```

Comment:
You can use `` to create multiline string in Golang.

Reply:
Thx, I find description without newline seems better.